torch/testing/_internal/common_distributed.py

```diff
@@ -569,6 +569,8 @@ def run_test(
             # Register signal handler to dump stack traces on FATALs.
             # Windows and MacOS do not support the signal handlers.
             torch._C._set_print_stack_traces_on_fatal_signal(True)
+        # Show full C++ stacktraces when a Python error originating from C++ is raised.
+        os.environ["TORCH_SHOW_CPP_STACKTRACES"] = "1"
```

Comment:
Is this env var also getting passed to all subprocesses?

Reply:
yep it should be passed to subprocesses, but this is actually already in the subprocess fn anyways